src/db/group.rs

```diff
@@ -4,7 +4,7 @@ use uuid::Uuid;
 
 use crate::db::{
     node::{Node, NodeIter, NodeRef, NodeRefMut},
-    Times,
+    CustomData, Times,
```

Comment:
@varjolintu I think this is a leftover from adding `CustomData` to the group struct. I'm guessing it will go away after a rebase?